src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -64,11 +66,13 @@
  * <li>
  * Property {@code excludeClassesRegexps} - Specify user-configured regular
  * expressions to ignore classes.
+ * Type is {@code Regexp}.
```

Comment:
`regexp` name should be used, we cannot use https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html as such type contains settings like (ignore case, ....) but we usually keep such parameters as separate properties.
so lets it be special type in checkstyle.

Reply:
@romani Just to confirm, only enumerations will display the full classpath? Other types like string, integers, collections, files, URLs should be the same as we display them on the xdoc?

Reply:
enumerations will display the full classpath? 

Yes.

>Other types like string, integers, collections, files, URLs should be the same as we display them on the xdoc?

Yes, because string and integers are standard types. We just name them by class name ( first letter is capital).
Collections will be bit renamed.
Other type stay the same as there is no class represtation for them.